src/main/java/ui/command/Command.java

```diff
@@ -0,0 +1,128 @@
+package ui.command;
+
+import data.TaskList;
+
+import java.util.ArrayList;
+
+/**
+ * @author Jiaaa-yang
+ *
+ * An abstract class to represent the commands
+ * that can be issued to a ChatBot.
+ */
+public abstract class Command {
```

Comment:
Do you think Parser is a better name for this class instead of Command? It seems that the main purpose of this class is to parse the inputs given by the users.

Reply:
Yeah the logic of parsing inputs could be placed in a `Parser` class instead to better separate my application logic.